csplat.y

```diff
@@ -55,6 +57,13 @@ when_stmt: WHEN L_PAREN add_exp R_PAREN LC stmts RC { printf("when_stmt -> WHEN
 | WHEN L_PAREN add_exp R_PAREN LC stmts RC ELSE LC stmts RC { printf("when_stmt -> WHEN L_PAREN exp R_PAREN LC stmts RC ELSE LC stmts RC\n");}
 | WHEN L_PAREN add_exp R_PAREN LC stmts RC ELSE when_stmt { printf("when_stmt -> WHEN L_PAREN exp R_PAREN LC stmts RC ELSE when_stmt\n");}
 
+whilst_stmt: WHILST exp LC stmt RC { printf("whilst_stmt -> WHILST exp LC stmt RC\n");}
+| WHILST exp LC RC { printf("whilst_stmt -> WHILST exp LC RC\n");}
```

Comment:
L_PAREN and R_PAREN will result in embedded parenthesis due to exp.
Ex: whilst (1 + 1 < 2) will not work but whilst ((1 + 1 < 2) will work if the rule is...
WHILST L_PAREN exp R_PAREN